build.sbt

```diff
@@ -16,7 +16,14 @@ addCommandAlias("gitSnapshots", ";set version in ThisBuild := git.gitDescribedVe
 val gh = GitHubSettings(org = "non", proj = "alleycats", publishOrg = "org.typelevel", license = mit)
 val devs = Seq(Dev("Erik Osheim", "non"))
 
-val vers = typelevel.versions ++ alleycats.versions
+val updates = Map(
+  "macro-compat" -> "1.1.0",
+  "export-hook" -> "1.1.0",
+  "simulacrum" -> "0.5.0"
+)
+val updatesSettings = Seq( resolvers += Resolver.sonatypeRepo("snapshots"))
```

Comment:
> All snapshot dependencies now replaced with releases, so this is ready to merge now.

Small point; As you now have no shapshot dependendencies, might we an idea to remove/comment out this line?